### PR TITLE
Change code styles to agreed settings

### PR DESCRIPTION
**Pull request by bnucci**
This should take care of #1

### CODE REVIEW

**Comment by radekkozak** [comment]
Looks ok, i'll give it a go on a live codebase if it will not do sth weird or unexpected with what we already got i think we'll be good to go

I'll try to clean the codebase hand in hand with RxNetwork project to see

**Review by radekkozak — CHANGES_REQUESTED** [review]
Ok, i think you forgot to rename the settings internally (within xml itself) so it doesn't appear in Intellij as Greyfox but rather as Square :)

File: configs/codestyles/GreyfoxAndroid.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <code_scheme name="SquareAndroid">
```
Comment (radekkozak) [inline configs/codestyles/GreyfoxAndroid.xml]: here

**Review by radekkozak — DISMISSED** [review]

File: configs/codestyles/GreyfoxAndroid.xml (hunk 1 above)
Comment (bnucci) [reply configs/codestyles/GreyfoxAndroid.xml]: done

**Review by radekkozak — APPROVED** [review]
I think we're good to 🚢 

In few rare instances we're gonna need to use Intellij `@formatter:on/off` (ie long generic params - don't think there is a setting for that anyway in code style)

**Pull request merged**